eth-types/src/geth_types.rs

```diff
@@ -158,6 +158,48 @@ pub fn get_rlp_unsigned(tx: &crate::Transaction) -> Vec<u8> {
     }
 }
 
+/// Get the RLP signed bytes
+/// NOTE: currently only used by curie upgrade calculating l1fee
+pub fn get_rlp_signed(tx: &crate::Transaction) -> Vec<u8> {
```

Comment:
@kunxian-xia is there any better method to do this?

Reply:
I think we can just use `tx.rlp()` instead.

Reply:
seems ok , will have a test.

Reply:
tested， will remove this new helper

Reply:
removed.